Technically -ffile-prefix-map is not OPTimisation flags,
but it's the easiest flags to be plugged in for all objects
--- a/Config.kmk
+++ b/Config.kmk
@@ -3721,7 +3721,7 @@ else
  # We should use -fomit-frame-pointer for GCC / -Oy for VCC.
  # -funwind-tables provides support for backtraces in gdb then.
  VBOX_GCC_FP     ?= -fno-omit-frame-pointer
- VBOX_GCC_OPT    ?= -O2 $(VBOX_GCC_mtune-generic)
+ VBOX_GCC_OPT    ?= -O2 $(VBOX_GCC_mtune-generic) -ffile-prefix-map=$(CURDIR)=.
  VBOX_VCC_FP     ?= -Oy-
  VBOX_VCC_OPT    ?= -O2
  VBOX_GCC_R0_FP  ?= -fno-omit-frame-pointer
